server/lib/template.js

```diff
@@ -62,10 +63,13 @@ function translate(translateCode, languageKey) {
    */
   function lookupTranslation(a, b) {
     var initialValue = !a;
+    let translate;
 
     if (initialValue) {
-      // lookup the value from the reference object
-      return en[b];
+      // translate to French if given else use English as default
+      translate = languageKey === 'fr' ? fr : en;
```

Comment:
If this was done outside of the `lookupTranslation` function it would not be run as many times - this is a small optimisation but could help for long keys.